crates/forge/src/main.rs

```diff
@@ -14,17 +15,21 @@ use forge::scarb::{
     paths_for_package, target_name_for_package, try_get_starknet_artifacts_path,
 };
 use std::process::{Command, Stdio};
+mod init;
 
 static PREDEPLOYED_CONTRACTS: Dir = include_dir!("crates/cheatnet/predeployed-contracts");
 
 #[derive(Parser, Debug)]
 #[command(version)]
 struct Args {
-    /// Name used to filter tests
-    test_name: Option<String>,
+    /// Name used to filter tests or project name during init
+    name: Option<String>,
```

Comment:
I wouldn't reuse this argument like this. It is confusing to have it do two things.

Instead we should do something like `init: Option<String>`, and handle it being `Some` or `None`.

Reply:
Nvm, your way is way cleaner @tkumor3, good job